test/test_perf.py

```diff
@@ -1,66 +0,0 @@
-# Copyright (c) Microsoft Corporation.
```

Comment:
Well the move to fixtures breaks this, and since we never run it anyway it's probably already broken, so seems like I might as well remove it.

If we care about it we should fix it and run it in CI.